core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -880,18 +1036,303 @@ export class LoggedInUser extends BaseUser {
   /**
    * Clicks the link with the text "discover more ways to get involved" on the Get Stated page.
    */
-  async clickDiscoverMoreWaysToGetInvolvedLinkInGetStartedPage(): Promise<void> {
+  async clickDiscoverMoreWaysToGetInvolvedLinkOnGetStartedPage(): Promise<void> {
     await this.page.waitForXPath(
       '//a[contains(text(),"discover more ways to get involved")]'
     );
-
     await Promise.all([
       this.page.waitForNavigation(),
       await this.clickOn('discover more ways to get involved'),
     ]);
 
     expect(this.page.url()).toBe(contactUrl);
   }
+
+  /**
+   * Clicks the link with the text "forum" on the Creator Guidelines page.
+   */
+  async clickForumLinkOnCreatorGuidelinesPage(): Promise<void> {
+    await this.page.waitForXPath('//a[contains(text(),"forum")]');
+    await Promise.all([this.page.waitForNavigation(), this.clickOn('forum')]);
+    await this.page.waitForNetworkIdle();
+
+    expect(this.page.url()).toBe(googleGroupsOppiaUrl);
+  }
+
+  /**
+   * Clicks the link with the text "Design Tips" on the Creator Guidelines page.
+   */
+  async clickDesignTipsLinkOnCreatorGuidelinesPage(): Promise<void> {
+    await this.page.waitForXPath('//a[contains(text(),"Design Tips")]');
+
+    await Promise.all([
+      this.page.waitForNavigation(),
+      await this.clickOn('Design Tips'),
+    ]);
+
+    expect(this.page.url()).toBe('http://oppia.github.io/#/DesignTips');
+  }
+
+  /**
+   * Clicks the link with the text "Create an Exploration" on the Creator Guidelines page.
+   */
+  async clickCreateAnExplorationLinkOnCreatorGuidelinesPage(): Promise<void> {
+    await this.page.waitForXPath(
+      '//a[contains(text(),"Create an Exploration")]'
+    );
+
+    await Promise.all([
+      this.page.waitForNavigation(),
+      await this.clickOn('Create an Exploration'),
+    ]);
+
+    expect(this.page.url()).toBe(
+      'http://localhost:8181/creator-dashboard?mode=create'
```

Comment:
Why not use the constants in test-constants.ts for the links? Ditto for the links below.